src/ISLogger.cpp

```diff
@@ -202,6 +234,29 @@ bool cISLogger::InitSaveCommon(eLogType logType, const string &directory, const
     return ISFileManager::PathIsDir(m_directory);
 }
 
+// (DEPRECATED)
+bool cISLogger::InitSave(eLogType logType, const string &directory, float driveUsageLimitPercent, uint32_t maxFileSize, bool useSubFolderTimestamp)
+{
+    sSaveOptions options;
+    options.logType                 = logType;
+    options.driveUsageLimitPercent  = driveUsageLimitPercent;
+    options.maxFileSize             = maxFileSize;
+    options.useSubFolderTimestamp   = useSubFolderTimestamp;
+    return InitSave(directory, options); 
+}
+
+// (DEPRECATED)
+bool cISLogger::InitSaveTimestamp(const string &timeStamp, const string &directory, const string &subDirectory, eLogType logType, float driveUsageLimitPercent, uint32_t maxFileSize, bool useSubFolderTimestamp)
```

Comment:
this should be "[[ deprecated ]]"  (which will generate a warning when this function is used).